pulse_actions/handlers/talos.py

```diff
@@ -0,0 +1,79 @@
+"""
+This module is for the following use case:
+
+ - Talos jobs:
+
+    * Trigger talos jobs twice if they are from PGO build.
+"""
+import logging
+
+from pulse_actions.utils.misc import filter_invalid_builders
+
+from mozci import query_jobs
+from mozci.errors import PushlogError
+from mozci.mozci import (
+    trigger_job
+)
+from mozci.sources import buildjson
+from requests.exceptions import ConnectionError
+
+LOG = logging.getLogger()
+
+def on_event(data, message, dry_run):
+    """Trigger talos jobs twice in PGO builds."""
+    # Cleaning mozci caches
+    buildjson.BUILDS_CACHE = {}
+    query_jobs.JOBS_CACHE = {}
+    payload = data["payload"]
+    status = payload["status"]
+    buildername = payload["buildername"]
+
+    if "pgo" in buildername and ("mozilla-inbound" in buildername or "fx-team" in buildername):
```

Comment:
Oh, I see that you're using the "talos" routing key in pulse. In that case you're assuming that it is a talos job.

Just in case, could you please add the check? Otherwise, we create the possibility of creating a pipeline of "garbage in; garbage out". This means that if the pulse stream gave us wrong builders on that routing we would be dealing with bad data.